src/main/java/teammates/ui/webapi/CreateAccountRequestAction.java

```diff
@@ -14,36 +15,30 @@
 class CreateAccountRequestAction extends AdminOnlyAction {
 
     @Override
-    public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOperationException {
+    public JsonResult execute()
+            throws InvalidHttpRequestBodyException, teammates.ui.webapi.InvalidOperationException {
```

Comment:
change `teammates.ui.webapi.InvalidOperationException` to `InvalidOperationException`